packages/compose/src/hooks/use-debounce/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * useDebounce hook
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+
+/**
+ * Use to debounce a value that is changing too frequently.
+ *
+ * Example usage:
+ *
```

Comment:
Including markdown code block notation with \`\`\`js will help highlight this text for documentation.

Reply:
Added 👍